app/src/routes/editor/WebviewArea/Frame.tsx

```diff
@@ -119,7 +121,7 @@ const Frame = observer(
         return (
             <div className="flex flex-col space-y-4">
                 <BrowserControls
-                    webviewRef={webviewRef}
+                    webviewRef={domReady ? webviewRef : null}
```

Comment:
Ah I see what you did. Apologies, I missed this. This makes sense and adds a level of security. I'd hate to have an error thrown needlessly.